.github/workflows/release.yaml

```diff
@@ -65,11 +65,12 @@ jobs:
 
       - name: Create Release Pull Request or Publish to NPM
         id: changesets
-        uses: changesets/action@v1
+        uses: FuelLabs/changesets-action@main
         with:
           publish: pnpm changeset:release
           commit: "ci(changesets): versioning packages"
           title: "ci(changesets): versioning packages"
+          createGithubReleases: aggregate
```

Comment:
It would be good to add the tagName and release name without it will get default values.

Reply:
I'm already setting this here https://github.com/FuelLabs/fuel-ui/blob/master/.changeset/config.json#L11-L14